### PR TITLE
docs: Added room documentation

### PR DESCRIPTION
**Pull request by dploeger**
Fixes: 

* godot-escoria/escoria-demo-game#1152

### CODE REVIEW

**Review by StraToN — APPROVED** [review]

File: getting_started/backgrounds.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+``ESCTerrain`` node in the scene: The ``scales`` parameter uses a texture.
+Darker areas of the texture result in a smaller sprite, while lighter areas 
+scale the sprite up. ``scale_min`` and ``scale_max`` limit the scaling.
+
+Lighting
+--------
+
+Another feature to make rooms more lively is to give them dynamic lighting,
+so that certain parts of the screen tint the character sprite to make it
+react to different light sources in the show picture.
```
Comment (BHSDuncan) [inline getting_started/backgrounds.rst]: Can you maybe re-word the ending of this paragraph? It isn't really clear what you mean by "light sources in the show picture".

**Review by BHSDuncan — CHANGES_REQUESTED** [review]
Please address the single comment left in the PR. Other than that, I've made some edits, and the rest looks good!

**Comment by dploeger** [comment]
Fixed, and I also included a fix in the step by step guide that came up on the Discord in the last days.

**Review by BHSDuncan — APPROVED** [review]
Made a small changing to phrasing but otherwise everything else looks good!

**Pull request merged**